admin/local/docker/compose/docker-compose.yml

```diff
@@ -98,6 +106,37 @@ services:
               source: volume_worker_0_mariadb_run
               target: /var/run/mysqld  # This is where the mariadb container puts the socket file
         network_mode: "service:worker-xrootd-0"
+    worker-wkr-0:
+        << : *worker-wkr
+        command: >
+          entrypoint worker-wkr
+          --db-uri mysql://qsmaster:CHANGEME2@127.0.0.1:3306
+          --db-admin-uri mysql://root:CHANGEME@127.0.0.1:3306
+          --vnid-config "@/usr/local/lib64/libreplica.so {{db_uri}}/qservw_worker 0 0"
+          --repl-instance-id qserv_proj
+          --repl-auth-key replauthkey
+          --repl-admin-auth-key=repladminauthkey
+          --repl-registry-host repl-registry
+          --repl-registry-port 25082
+          --results-dirname /qserv/data/results
+          --mysql-monitor-password CHANGEME_MONITOR
+          --log-cfg-file=/config-etc/log/log-worker-xrootd.cnf
```

Comment:
If you remove `xrootd` then it makes sense to call the config file as simply as `log-worker.cnf`.